src/components/input-addon/index.js

```diff
@@ -0,0 +1,57 @@
+import { nativeVlElement, define } from "../../utils/core/";
+
+/**
+ * Gebruik de input addon mixin in combinatie met een input addon elementen.
+ * @mixin vlInputAddonElement
+ *
+ * @param {Object} SuperClass
+ * @return {Object} SuperClass
+ *
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-input-addon/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-input-addon/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-input-addon.html|Demo}
+ */
+export const vlInputAddonElement = (SuperClass) => {
```

Comment:
Mixins best in een aparte mixin folder, om consistent te blijven

Reply:
FIxed.